### ISSUE TITLE
xeft isn't finding the dynamic's module path

### ISSUE THREAD

**Issue by KonstantinDjairo** [casouri/xeft#31]
there could be a variable to set the path to the .so file, would prevent this

**Comment by KonstantinDjairo** [comment]
alright, it seems that xeft wont find the module if you dont have the folder in the path
```(add-to-list 'load-path "/path/to/directory/")```
and also it will look for a specific path "~/.emacs.d/xeft/" , this could be solved looking at the code and correcting to the actual path that im using

**Comment by casouri** [comment]
Yeah, in general, dynamic modules are like normal elisp files, so `load-path` applies to them too. I should add some explanation in the README. Thanks!
